### ISSUE TITLE
Update help file messages

### ISSUE THREAD

**Issue by Eudaemonea** [tastybento/askyblock#120]
If you could, please update the help files info to not give message on screen to use /island help for info. Should be /ai help instead for those of us running both acid island and skyblock. Confusing to the players.

thanks! <3

**Comment by tastybento** [comment]
Done in the latest version.

**Comment by Eudaemonea** [comment]
Thanks a bunch.

I did have one other thing someone brought to our attention that's pretty
minor (for us anyway).

If someone logs out while on a ladder, and logs back in they have /fly
turned on sometimes.  We saw it with skyblock too but not sure if that's
something just glitching on our end or plugin related.  wasn't ever a big
deal to me but figured i'd mention it just in case

and btw... you've dazzled my lady staff.  they're in love with your accent
after watching your acid island video lol
